lerna.json

```diff
@@ -4,7 +4,8 @@
     "cloud/*",
     "client/*",
     "examples/*",
-    "demo/*"
+    "demo/*",
+    "utilities/*"
```

Comment:
IMHO let's put that initially to cloud. 
As we categorize our modules for platform (server,client)